content/en/platform/corda/4.8/open-source/key-concepts-ledger.md

```diff
@@ -20,52 +20,50 @@ title: Ledger
-
-**What does that mean?**
-
-* There are no omniscient peers who can see everything on the ledger.
-* Each node sees their own facts, and the facts that they share with others.
 
-For example, Alice and Demi don’t share any facts, so they each see a completely different set of shared facts.
+In the diagram, facts 1 and 7 are known by both Alice and Bob. Alice only shares facts with Bob, Alice doesn't share
+any facts with Carl, Demi, or Ed.
 
-## Shared Facts
+Each node only sees a subset of facts—their own facts and those that they share with others. No single node can view
+the ledger in its entirety. For example, in the diagram Alice and Demi don’t share any facts, so they see a
+completely different set of facts to each other.
 
-In the previous diagram, we could see that although Carl, Demi and Ed are aware of shared fact 3, **Alice and Bob are not**.
+## Shared facts
 
-﻿On Corda, there is no central or general ledger operating with agency on ﻿behalf of all of the nodes on the network. Instead, each node on the network maintains its own vault containing all of its known facts.
+On Corda, there is no central or general agent which operates on behalf of all of the nodes on the network. Instead,
+each node maintains its own vault which contains all of its known facts.
 
-You can think of this vault as being a database or simple table.
+You can think of a vault as being a database or simple table. In this diagram, facts 1 and 7 appear on both Alice's
+vault and Bob's vault, and are therefore shared facts:
 
 {{< figure alt="ledger table" width=80% zoom="/en/images/ledger-table.png" >}}
-Where the rows are shared between nodes (facts 1 and 7 in this example), these are shared facts.
 
-Corda guarantees that whenever one of these facts is shared by multiple nodes on the network, it evolves in lockstep in the database of every node that is aware of it. This means that Alice and Bob will both see an **exactly identical version** of shared facts 1 and 7.
+Corda guarantees that when a fact is shared across multiple nodes on the network, it evolves in lockstep in each of
+these nodes' vaults. This means that Alice and Bob will both see an *exactly identical version* of shared facts 1 and 7.
 
-**Note:** Not all on-ledger facts are shared between peers. For example, Alice's fact 11 is not shared with Bob. Fact 11 could, in fact, not be shared with any other node at all. If this is the case, it is deemed a unilateral fact.
+On-ledger facts don't have to be shared between nodes. For example, fact 11 in Alice's vault is not shared with Bob.
+Facts that are not shared are *unilateral facts*.
 
-**Note:** Although there is no central ledger, it is possible to broadcast a basic fact to all participants should you wish to. You would do this by using the network map service to loop over all parties.
+{{< note >}}
+Although there is no central ledger, you can broadcast a basic fact to all nodes of a network using the network map
```

Comment:
I might give an example/use case for this to further clarify what that means, and also link to the network map page.